CODEOWNERS

```diff
@@ -1 +1 @@
-* @dagguh @kema2000 @jkim2-atlassian @mk-atlassian @mminns
+* @atlassian/jpt @jkim2-atlassian @mminns
```

Comment:
Interesting, I remember handing it over to Guardians. @pczuj did something change in that regard?

Reply:
I'm not sure what I'm looking at, but in terms of HWR you recall correctly that there was a step introduced in the release process to generate the HWR and the release process is executed by the release manager - a role provided by Guardians. I don't think we ever had an ownership of the code behind HWR.

Reply:
Let's discuss this offline.